x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -39,9 +39,12 @@ export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
   const { isLoading: isLoadingAlertFeatureIds, data: alertFeatureIds } =
     useGetFeatureIds(alertRegistrationContexts);
 
+  const configId =
+    caseData.owner === SECURITY_SOLUTION_OWNER ? `${caseData.owner}-case` : caseData.owner;
```

Comment:
To avoid having specific logic only for Security solution I think it is better if we prefix `cases` to all tables. For example `const configId = `cases-${caseData.owner}`